### PR TITLE
Extend result codes

### PR DESCRIPTION
**Pull request by VVeremjova**
Now ATF gets next codes:

Code 0 - all tests in the script were executed, have "PASS" result (overall status "PASSED")
Code 1 - not all test were executed - unexpected SDL disconnect, ATF's PANIC, etc. (overall status "ABORTED")
Code 2 - all tests in the script were executed, at least one test have "FAIL" result (overall status "FAILED")
Code 3 - ATF was started with wrong arguments

Related to [APPLINK-29374](https://adc.luxoft.com/jira/browse/APPLINK-29374)

### CODE REVIEW

**Comment by VVeremjova** [comment]
Please review @AByzhynar, @dcherniev, @Kozoriz, @dtrunov, @pvvasilev

File: modules/atf/stdlib/argument_parser.lua [hunk 1]
```diff
@@ -43,13 +43,13 @@ local function errmsg (msg)
   if msg:match ("%.$") == nil then msg = msg .. "." end
   print (prog .. ": error: " .. msg .. "\n")
   print (prog .. ": Try '" .. prog .. " --help' for help.")
-  quit(2)
+  quit(3)
```
Comment (Kozoriz) [inline modules/atf/stdlib/argument_parser.lua]: @VVeremjova , what changes? can you move this to some var? Or create some enum of consts with exit codes.

Comment (VVeremjova) [inline modules/atf/stdlib/argument_parser.lua]: @Kozoriz See 117cb52

File: modules/schema_validation.lua [hunk 2]
```diff
@@ -193,8 +204,15 @@ function module.mt.__index:CompareType(data_elem, schemaElem, isArray, nameofPar
     if(self.schema.interface[interface_name].enum[complex_elem_name][data_elem] ~= nil) then 
       return true
     end 
+    --Enum element can be sended as number, so if it includes to data elements, we will set it as acceptable
```
Comment (AByzhynar) [inline modules/schema_validation.lua]: @VVeremjova `sended` - is wrong. `sent` should be used, What does it mean : `if it includes to data elements`?

Comment (VVeremjova) [inline modules/schema_validation.lua]: @AByzhynar   I meant if number is less than count of elements in enum, then we assume it belongs to enum

**Pull request closed without merging**